crates/phactory/src/system/mod.rs

```diff
@@ -845,7 +794,7 @@ impl<Platform: pal::Platform> System<Platform> {
     ) {
         if !origin.is_gatekeeper() {
             error!("Invalid origin {:?} sent a {:?}", origin, event);
-            return;
+            panic!("System state poisoned");
```

Comment:
We should ignore. Generally anyone can send any message. The messaging system only guarantee the origin is correct. So it's up to the receiver to check the authority of the message sender.

Imagine a malicious user can just deploy an ink contract, and write a function to send a message to this topic. Then it's easy to crash any worker

Reply:
Please address this problem